packages/svelte2tsx/svelte-jsx.d.ts

```diff
@@ -250,7 +291,65 @@
       // Transition Events
       ontransitionend?: TransitionEventHandler<T>;
       ontransitionendcapture?: TransitionEventHandler<T>;
-    }
+
+      // HashChange Events
```

Comment:
Can you add a new interface for the window specific events? like `WindowEventHandlersAttributes` and make the `SvelteWindowProps` `extends` it, Or you can just move these events To `SvelteWindowProps` interface. These events don't apply to `HTMLElement`. 
Also, I think we don't need capture-suffix events. The existing are adopted from the react's definition.

Reply:
I've removed the capture suffix and and moved the events into the SvelteWindowProps interface